src/entities/User.ts

```diff
@@ -66,9 +68,13 @@ export class User {
 	@JoinColumn()
 	public profile?: Profile;
 
+	@OneToOne(() => Report, report => report.reportedUser, { nullable: true, eager: true, cascade: true })
+	@JoinColumn()
+	public report?: Report;
+
 	public toJSON(): APIUser {
-		const { id, forename, surname, accountStatus, accountType, profile } = this;
-		return { id, forename, surname, accountStatus, accountType, profile: profile?.toJSON() };
+		const { id, forename, surname, accountStatus, accountType, profile, report } = this;
+		return { id, forename, surname, accountStatus, accountType, profile: profile?.toJSON(), report: report?.toJSON() };
```

Comment:
As discussed above with reports not really needing to be fetched when getting a user's info (if anything it should be hidden!) can you remove this from the toJSON?